test/integration/basic_test.go

```diff
@@ -99,6 +99,18 @@ func (m *Minishift) executingOcCommand(command string) error {
 	return nil
 }
 
+func (m *Minishift) executingOcCommandSucceeds(command string) error {
+	err := m.executingOcCommand(command)
+	if err != nil {
+		return err
+	}
+	if lastCommandOutput.ExitCode != 0 && len(lastCommandOutput.StdErr) != 0 {
```

Comment:
&&? I think it should be || here. If either of the two occurs the test should fail.